test/e2e/metamask-ui.spec.js

```diff
@@ -191,12 +191,9 @@ describe('MetaMask', function () {
 
     it('imports Secret Recovery Phrase', async function () {
       const restoreSeedLink = await driver.findClickableElement(
-        '.unlock-page__link--import',
-      );
-      assert.equal(
-        await restoreSeedLink.getText(),
-        'import using Secret Recovery Phrase',
+        '.unlock-page__link',
       );
+      assert.equal(await restoreSeedLink.getText(), 'Forgot Password?');
```

Comment:
Does this test still work? I thought we are using sentence case here? Shouldn't it be `'Forgot password?'`

Reply:
Currently, the forgot password link from the Unlock page is `Forgot Password?`, I am waiting on @mcmire comment on changing the case.

Reply:
Ah ok. That makes sense. I did a search in the PR and couldn't find the `Forgot Password` instance. My bad